executor/exeerrors/errors.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package executor
```

Comment:
In fact, We have some errors defined at ```util/dbterror``` such as DDL.

Reply:
https://github.com/pingcap/tidb/pull/42075/commits/80a4048477469869714db3f2ec5645bd0e114641